data-raw/watershed/cow/cow_creek.rmd

```diff
@@ -16,16 +16,13 @@ library(cvpiaHabitat)
 Instream habitat suitability relationships provided for the Voluntary Agreement process.
```

Comment:
we don't have new data coming for cow creek, so change to none planed

Reply:
done